collectors/python.d.plugin/rabbitmq/rabbitmq.chart.py

```diff
@@ -104,6 +112,18 @@
             ['object_totals_exchanges', 'exchanges', 'absolute']
         ]
     },
+    'churn_rates': {
+        'options': [None, 'Churn Rates', 'changes/s', 'overview', 'rabbitmq.churn_rates', 'line'],
```

Comment:
Hi. I think units should be `operations/s`.

Reply:
And it could be better to split it into the 3 charts:
 - connection churn rates
 - channel churn rates
 - queue churn rates

What do you think?

Reply:
Yes, operations/s is a better unit.
Splitting to 3 charts would be fine.